src/main/scala/vectorpipe/vectortile/Pipeline.scala

```diff
@@ -1,151 +1,194 @@
-   * String-typed column of the name contained in [[layerName]] to indicate
-   * which layer a geometry belongs in.
-   */
+    * Select geometries for display at a given zoom level.
+    *
+    * VectorPipe allows geometries to be carried up the pyramid for display at a
+    * later time.  This function is used to choose the elements that will be
+    * displayed at the target zoom level.  This is useful for implementing a
+    * display schema, or for example, selection based on prominence.  VectorPipe
+    * will add a column of Array[SpatialKey] in the column with the name given by
+    * `keyColumn`.  These keys identify the layout tiles which intersect each
+    * geometry at the current zoom level.  This information may be useful when
+    * deciding which geometries to display.
+    *
+    * If [[vectorpipe.vectortile.Pipeline#layerMultiplicity layerMultiplicity]]
+    * is true, the resulting DataFrame must supply a String-typed column of the name
+    * contained in [[vectorpipe.vectortile.Pipeline#geometryColumn geometryColumn]]
+    * to indicate which layer a geometry belongs in.
+    */
   def select(input: DataFrame, targetZoom: Int, keyColumn: String): DataFrame = input
 
   /**
-   * Clip geometries prior to writing to vector tiles.
-   *
-   * It may be desirable to carry only the portion of a geometry that intersects
-   * a given vector tile to keep down memory usage.  This function can be used
-   * to implement such schemes.
-   *
-   * Basic (non-no-op) clipping functions can be found in [[Clipping]].
-   */
+    * Clip geometries prior to writing to vector tiles.
+    *
+    * It may be desirable to carry only the portion of a geometry that intersects
+    * a given vector tile to keep down memory usage.  This function can be used
+    * to implement such schemes.
+    *
+    * Basic (non-no-op) clipping functions can be found in
+    * [[vectorpipe.vectortile.Clipping Clipping]].
+    */
   def clip(geom: Geometry, key: SpatialKey, layoutLevel: LayoutLevel): Geometry = geom
 
   /**
-   * Convert table rows to output features.
-   *
-   * A straightforward conversion from a table row to a geometric feature.  The
-   * data carried by the feature are stored as entries in a Map[String, Value].
-   * See [[geotrellis.vectortile.Value]] for details.
-   */
+    * Convert table rows to output features.
+    *
+    * A straightforward conversion from a table row to a geometric feature.  The
+    * data carried by the feature are stored as entries in a Map[String, Value].
+    * See [[geotrellis.vectortile.Value]] for details.
+    */
   def pack(row: Row, zoom: Int): VectorTileFeature[Geometry] =
     Feature(row.getAs[Geometry](geometryColumn), Map.empty)
+
+  /**
+    * Receive the final RDD of generated vector tiles for a zoom level
+    * to finish any additional processing or handling such as persisting to a datastore.
+    */
+  def finalize(vectorTiles: RDD[(SpatialKey, VectorTile)], zoom: Int): Unit
```

Comment:
I couldn't think of a better name for this method. Alternatives I was thinking of was: `finish`, `complete`, `done`

Reply:
`finalize` works for me